Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -151,10 +149,10 @@ namespace Hazel {
 	void WindowsWindow::Shutdown()
 	{
 		glfwDestroyWindow(m_Window);
+		--s_GLFWWindowCount;
 
-		if (--s_GLFWWindowCount == 0)
+		if (s_GLFWWindowCount == 0)
 		{
-			HZ_CORE_INFO("Terminating GLFW");
```

Comment:
Same as with the init message.

Reply:
Cherno actually specifically said he wanted these logs gone, they aren't randomly being removed.

Reply:
See same as the init message https://github.com/TheCherno/Hazel/pull/165#discussion_r344480639. Marking as resolved